src/main/resources/explainer/article/index.scala.html

```diff
@@ -3,3 +3,10 @@
 )
 
 
+<div class="atom--explainer"> 
```

Comment:
Let's just add the `atom` class for good measure

Reply:
sure so like `<div class="atom atom--explainer"> `

Reply:
oui